Neos.Cache/Classes/Backend/RedisBackend.php

```diff
@@ -251,6 +260,23 @@ private function buildKey(string $identifier): string
         return $this->cacheIdentifier . ':' . $identifier;
     }
 
+    /**
+     * Calculate the max lifetime for a tag
+     *
+     * @param string $tag
+     * @param int $lifetime
+     * @return int
+     */
+    private function calculateExpires($tag, $lifetime)
+    {
+        $ttl = $this->redis->ttl($tag);
+        if ($ttl === -1 || $lifetime === self::UNLIMITED_LIFETIME) {
+            return self::UNLIMITED_LIFETIME;
+        } else {
+            return max($ttl, $lifetime);
+        }
```

Comment:
could have a typed return, also else statement is unnecessary

Reply:
Sorry I don't get it. Why do you think the else statement is unnecessary.
Would you prefer `return ($ttl === -1 || $lifetime === self::UNLIMITED_LIFETIME) ? self::UNLIMITED_LIFETIME : max($ttl, $lifetime)`. Sorry for using the wrong branch will change that.

Reply:
just meant the `} else {` part is not needed.. it returns early